tests/e2e/ansible/start_docker_registry.yaml

```diff
@@ -19,19 +19,15 @@
         state: present
       retries: 3
       delay: 10
-    # The docker pip is needed by the docker_container ansible module itself.
-    - name: Install docker pip
+    # The docker and requests pip packages are required by the docker_container ansible module itself.
+    # The requests package is pinned to a version less than 2.32 to avoid a bug
+    # at https://github.com/docker/docker-py/issues/3256
+    - name: Install docker and requests pip packages
       pip:
-        name: docker
-    # Due to a bug in requests 2.29.0 we need to downgrade it
-    # see https://github.com/docker/docker-py/issues/3113 
-    - name: Downgrade requests
-      shell: |
-        actual_version="$(pip3 show requests | grep Version | cut -d: -f2 | xargs)"
-        version_ge="$(echo -e "2.29.0\n$actual_version" | sort -V | head -1)"
```

Comment:
The other day I was looking for this exact line and I couldn't find. This is a good way to compare semver numbers in bash. So long lovely code, so long!